src/state/opamSysPoll.ml

```diff
@@ -10,6 +10,10 @@
 
 open OpamStd.Option.Op
 
+let make_lazy f =
+  let v = Lazy.from_fun f in
+  fun () -> Lazy.force v
+
```

Comment:
Sorry - looking at how it's then gone below, I this could be dropped and just use `Lazy.from_fun` below...